app/util/views/Summary.scala

```diff
@@ -38,6 +40,27 @@ case class AssetSummary(template: String) extends Summarizer[Asset] {
       case false => asset.getType().label
     }
   }
+  def getStatusLabel(asset: Asset): String = {
+    val state_string = asset.getState match {
+      case Some(st) => ":%s" format st.name
+      case None => ""
+    }
+    val state_descr = asset.getState match {
+      case Some(st) => " - %s" format st.description
+      case None => ""
+    }
+    val badge_class = asset.getStatus.name match {
+      case "Allocated"                    => "label-primary"
+      case "Cancelled" | "Decommissioned" => "label-default"
+      case "Maintenance"                  => "label-danger"
+      case "Incomplete" | "New"           => "label-warning"
+      case "Provisioned" | "Provisioning" => "label-info"
+      case "Unallocated"                  => "label-success"
+      case _                              => "label-default"
+    }
```

Comment:
Agreed. It would be way more hassle than its worth to break colors out into config files. I dont like it either, but... I didnt want to spend all day doing this. :)